### PR TITLE
DCOS cassandra

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: roles/frameworks/templates/cassandra-config.j2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (tayzlor) [inline roles/frameworks/templates/cassandra-config.j2]: should we expose the other config vars here or just leave it for now? https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/C/cassandra/0/config.json

File: roles/frameworks/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+# defaults file for frameworks
+frameworks_dcos_cli_image: capgemini/dcos-cli
+frameworks_zk_master_peers: "zk://{{ zookeeper_peers_nodes }}/mesos"
+frameworks_mesos_master_url: "http://{{ ansible_ssh_host }}:5050"
+frameworks_marathon_url: "http://{{ ansible_ssh_host }}:8080"
```
Comment (tayzlor) [inline roles/frameworks/defaults/main.yml]: should this use ```{{ marathon_port }}```

File: roles/frameworks/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+# defaults file for frameworks
+frameworks_dcos_cli_image: capgemini/dcos-cli
+frameworks_zk_master_peers: "zk://{{ zookeeper_peers_nodes }}/mesos"
+frameworks_mesos_master_url: "http://{{ ansible_ssh_host }}:5050"
```
Comment (tayzlor) [inline roles/frameworks/defaults/main.yml]: Should use ```{{ mesos_master_port }}```

**Comment by tayzlor** [comment]
:+1: really like this PR!

**Comment by wallies** [comment]
very nice :+1:

File: roles/frameworks/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+# defaults file for frameworks
+frameworks_dcos_cli_image: capgemini/dcos-cli
+frameworks_zk_master_peers: "zk://{{ zookeeper_peers_nodes }}/mesos"
+frameworks_mesos_master_url: "http://{{ ansible_ssh_host }}:{{ mesos_master_port }}"
+frameworks_marathon_url: "http://{{ ansible_ssh_host }}:{{ marathon_port }}"
+
+frameworks_cassandra_enabled: true
```
Comment (wallies) [inline roles/frameworks/defaults/main.yml]: are we going to enable this by default. all frameworks except marathon should be disabled by default i think

Comment (enxebre) [inline roles/frameworks/defaults/main.yml]: ups no, was testing

File: roles/frameworks/templates/cassandra-config.j2 (hunk 1 above)
Comment (wallies) [inline roles/frameworks/templates/cassandra-config.j2]: even expose the minimum required configuration values would be nice with the default in the config.json e.g.
"framework","cluster-name","zk-timeout-ms","node-count","seed-count","health-check-interval-seconds","bootstrap-grace-time-seconds","data-directory","resources"

Comment (tayzlor) [inline roles/frameworks/templates/cassandra-config.j2]: can be a follow-up issue

**Pull request merged**